src/Root.jsx

```diff
@@ -13,7 +13,7 @@ export default class Root extends React.Component {
 
         this.state = {
             searchResults: undefined,
-            movieData: undefined
+            movieDetails: undefined
```

Comment:
Will be better define a default value for it instead of `undefined`, maybe an object with properties or anything related. For searchResults could be an empty array also.